app/src/org/commcare/activities/connect/ConnectIdManager.java

```diff
@@ -529,6 +542,13 @@ public static void handleFinishedActivity(int requestCode, int resultCode, Inten
                     }
                 }
             }
+            case CONNECT_BIOMETRIC_ENROLL_FAIL -> {
+                nextRequestCode = ConnectIdTask.CONNECT_REGISTRATION_CONFIGURE_BIOMETRICS;
+                if(success) {
+                    //Go to settings
+                    launchSecuritySettings = true;
```

Comment:
why not fire the settings screen directly from here ?

Reply:
The call to continueWorkflow() launches the next Connect activity (in this case it re-launches ConnectIdVerificationActivity after showing the message activity). I still want that activity to launch again, but then I want the settings activity to launch on top of it therefore I need it to be launched second.

Reply:
The navigation here seems a bit complex to me. I would think a more simplified workflow here would be for Verification activity to show the error and settings screen button directly instead of us going back and forth with verification and message activity.